CrowdJustice/legl-dev
# Add install command

### What does this PR do?
Add install command to install packages to the containers and to update legl-dev itself
### How do you test it?
- `pip install git+https://github.com/CrowdJustice/legl-dev.git@add-install-command`
- try installing a pip package `legl-dev install --pip example`
- the package should be installed to you `backend` container and show up in `requirements.txt`
- try installing a yarn package `legl-dev install --yarn example`
- the package should be installed to you `frontend` container and show up in `package.json`
- try upgrading legl-dev `legl-dev install --upgrade  --self`

### What should the release note say?



<release-notes>Add install command to install packages to the containers and to update legl-dev itself
<release-notes>
